doc-content/enterprise-only/openshift/environment-authoring-con.adoc

```diff
@@ -1,16 +1,43 @@
 
-An HA authoring environment contains several pods. Both {CENTRAL} and {KIE_SERVER} are provided in scalable pods that can run in parallel and share persistent storage. The database is provided by a separate pod. Use a high-availability authoring environment to provide maximum reliability and responsiveness, especially if several users are involved in authoring at the same time.
+For {PRODUCT}, the procedures to deploy an authoring environment and a managed server environment are the same. You must first deploy an authoring environment template, consisting of {CENTRAL} and one {KIE_SERVER}. Then, if necessary, you can deploy additional {KIE_SERVER} templates in the same namespace to create a managed server environment with multiple {KIE_SERVERS}.
+endif::DM[]
 
-You can also deploy additional managed or immutable {KIE_SERVERS}, if required. {CENTRAL} can automatically discover any {KIE_SERVERS} in the same namespace, including immutable {KIE_SERVERS} and managed {KIE_SERVERS}. This feature requires the `OpenShiftStartupStrategy` setting, which is enabled for all {KIE_SERVERS} except those deployed in a fixed managed infrastructure. For instructions about deploying managed {KIE_SERVERS} with the `OpenShiftStartupStrategy` setting enabled, see {URL_DEPLOYING_MANAGED_FREEFORM_ON_OPENSHIFT}[_{DEPLOYING_MANAGED_FREEFORM_ON_OPENSHIFT}_]. For instructions about deploying immutable {KIE_SERVERS}, see {URL_DEPLOYING_IMMUTABLE_ON_OPENSHIFT}[_{DEPLOYING_IMMUTABLE_ON_OPENSHIFT}_].
+Depending on your needs, you can deploy either a single authoring environment template or a high-availability (HA) authoring environment template.
+
+A single authoring environment contains two pods. One of the pods runs {CENTRAL}, the other runs {KIE_SERVER}. 
+ifdef::PAM[The {KIE_SERVER} includes an embedded H2 database engine.]
+This type of environment uses the least possible amount of resources. 
+
+An HA authoring environment contains several pods. Both {CENTRAL} and {KIE_SERVER} are provided in scalable pods that can run in parallel and share persistent storage. 
+ifdef::PAM[The database is provided by a separate pod.]
+Additional pods also contain the AMQ and Data Grid servers that enable high-availability functionality in {CENTRAL}. Use a high-availability authoring environment to provide maximum reliability and responsiveness, especially if several users are involved in authoring at the same time.
+
+In the current version of {PRODUCT}, an HA authoring environment is supported with certain limitations:
+
+* If one user is deploying a service to the {KIE_SERVER}, another user will not be able to deploy a service to the same {KIE_SERVER} until the first deployment completes.
```

Comment:
This limitation is not only for HA but for whole OpenShiftStartupStrategy

Reply:
HA is when the user is likely to feel it.